scripts/update-git-platinum.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+## This is an update script for /data/git-platinum. It's meant to be executed
+## from the root of the repo.
```

Comment:
We could check that we're at the root by doing:

```bash
BASE=$(basename pwd)

if [ "${BASE}" -ne "radicle-surf" ]
then
   echo "this script should be run from the root of radicle-surf"
   exit 1
fi
```

Or something along those lines :)

Reply:
That's excellent. I'll give it a try. I was wondering if that's possible, but my bash powers are honestly fairly weak ;)

Reply:
Mine aren't much better, but I sure can google! :P